SPECS/librsvg2/librsvg2.spec

```diff
@@ -1,64 +1,91 @@
+# https://github.com/rust-lang/rust/issues/47714
+%undefine _strict_symbol_defs_build
+
 # We want verbose builds
 %global _configure_disable_silent_rules 1
 # Use bundled deps as we don't ship the exact right versions for all the
 # required rust libraries
 %global bundled_rust_deps 1
 %global cairo_version 1.16.0
+
 Summary:        An SVG library based on cairo
 Name:           librsvg2
-Version:        2.48.9
-Release:        4%{?dist}
+Version:        2.50.3
+Release:        2%{?dist}
 License:        LGPLv2+
 Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL:            https://wiki.gnome.org/Projects/LibRsvg
-Source0:        https://download.gnome.org/sources/librsvg/2.48/librsvg-%{version}.tar.xz
+Source0:        https://download.gnome.org/sources/librsvg/2.50/librsvg-%{version}.tar.xz
 BuildRequires:  cairo-devel >= %{cairo_version}
 BuildRequires:  cairo-gobject-devel >= %{cairo_version}
-BuildRequires:  cargo
 BuildRequires:  chrpath
+BuildRequires:  fontconfig-devel
 BuildRequires:  gcc
-BuildRequires:  git-core
+BuildRequires:  gdk-pixbuf2-devel
+BuildRequires:  git
 BuildRequires:  gobject-introspection-devel
-BuildRequires:  pkg-config
-BuildRequires:  rust
+BuildRequires:  harfbuzz-devel >= 2.0.0
+BuildRequires:  make
+BuildRequires:  pkgconfig
 BuildRequires:  vala
-BuildRequires:  pkgconfig(fontconfig)
-BuildRequires:  pkgconfig(gdk-pixbuf-2.0)
+BuildRequires:  vala-devel
+BuildRequires:  vala-tools
+BuildRequires:  pkgconfig(cairo-png) >= %{cairo_version}
 BuildRequires:  pkgconfig(gio-2.0)
 BuildRequires:  pkgconfig(gio-unix-2.0)
 BuildRequires:  pkgconfig(glib-2.0)
 BuildRequires:  pkgconfig(gthread-2.0)
 BuildRequires:  pkgconfig(libxml-2.0)
 BuildRequires:  pkgconfig(pangocairo)
 BuildRequires:  pkgconfig(pangoft2)
-Requires:       cairo >= %{cairo_version}
-Requires:       cairo-gobject >= %{cairo_version}
+Requires:       cairo%{?_isa} >= %{cairo_version}
+Requires:       cairo-gobject%{?_isa} >= %{cairo_version}
 # We install a gdk-pixbuf svg loader
-Requires:       gdk-pixbuf2
-# https://github.com/rust-lang/rust/issues/47714
-%undefine _strict_symbol_defs_build
+Requires:       gdk-pixbuf2%{?_isa}
+%if 0%{?bundled_rust_deps}
```

Comment:
Thanks for removing the macros but that was only half of the work. Please also remove the parts of the spec, which will never get executed because the macros are not there.